setup.cfg

```diff
@@ -146,6 +146,8 @@ exclude =
 [options.entry_points]
 console_scripts =
     dvc = dvc.cli:main
+fsspec.specs =
+    dvc = dvc.api:DvcFileSystem
```

Comment:
This will automatically add to `fsspec`'s registry, so you can create dvcfs in following ways if you have dvc installed:
```
from fsspec import filesystem

fs = filesystem("dvc")
fs = filesystem("dvc", url="/path/to/repo")
fs = filesystem("dvc", url="https://github.com/iterative/dataset-registry", rev="main")
fs = filesystem("dvc", repo=repo_instance)
```